Retrie/ExactPrint.hs

```diff
@@ -256,9 +258,9 @@ swapEntryDPT a b = do
 
 -- Compatibility module with ghc-exactprint
 
-parseContentNoFixity :: Parsers.LibDir -> FilePath -> String -> IO AnnotatedModule
-parseContentNoFixity libdir fp str = do
-  r <- Parsers.parseModuleFromString libdir fp str
+parseContentNoFixity :: [Extension] -> Parsers.LibDir -> FilePath -> String -> IO AnnotatedModule
```

Comment:
Most of this PR is just threading `[Extension]` through the places that need it.